modules/rosa-create-objects.adoc

```diff
@@ -116,14 +116,20 @@ $ rosa create cluster --cluster=<cluster_name> | <cluster_id> [arguments]
 |--private
 |Restricts master API endpoint and application routes to direct, private connectivity.
 
+|--private-link
+| Specifies to use AWS PrivateLink to provide private connectivity between VPCs and services. The `--subnet-ids` argument is required when using `--private-link`.
+
 |--region
 |The AWS region (string) where your worker pool will be located. This argument overrides the `AWS_REGION` environment variable.
 
 |--service-cidr
 |Block of IP addresses (ipNet) for services. Example: `172.30.0.0/16`
 
 |--subnet-ids
-|The subnet IDs (string) to use when installing the cluster. Subnet IDs must be in pairs with one private subnet ID and one public subnet ID per availability zone. Subnets are comma-delimited. Example: `--subnet-ids=subnet-1,subnet-2`. Leave the value empty for installer-provisioned subnet IDs.
+|The subnet IDs (string) to use when installing the cluster. Subnet IDs must be in pairs with one private subnet ID and one public subnet ID per availability zone. Subnets are comma-delimited. Example: `--subnet-ids=subnet-1,subnet-2`. Leave the value empty for installer-provisioned subnet IDs. 
+
+
+The `--subnet-ids` argument is required when using `--private-links`.
```

Comment:
It's better clarify that only 1 private subnet per zone is allowed when --private-links set